randovania/games/prime2/json_data/Torvus Bog.txt

```diff
@@ -698,6 +698,9 @@ Extra - in_dark_aether: False
                   # https://www.youtube.com/watch?v=pJlUeNML-vo No Space Jump
                   Bomb Space Jump (Advanced)
                   Space Jump Boots and Bomb Space Jump (Intermediate)
+          All of the following:
+              # https://youtu.be/RSIbffrfJaE
```

Comment:
Video cuts off before reaching the ledge. An updated video that shows this would be nice.

Reply:
I uploaded a new video, didn't realize that it got cut off, whoops!